### DISCUSSION TITLE
Kong Rate limit and other consumer specific plugins stopped working

### DISCUSSION THREAD

**Discussion by subalalithafl** [Kong/kong#8741] [Help] [answered]
I am using Kong version 2.6 and i have key-auth enabled for all consumers. I have also added rate-limit plugin for all users. This was working fine. Rate was limited to 5 calls per sec etc.

But suddenly all plugins configured at consumer level stopped working. Plugins are Rate-limit and statsd plugin. Not sure what is the cause here. Any pointer on how to debug this is much appreciated.

I am not using any custom plugins, and i did not change anything major.

**Comment by subalalithafl** [comment]
I have upgraded the Kong version to 2.8.1 and imported the old db into a new one but still my rate-limit plugin is not working. Can someone help here please?

**Comment by ADD-SP** [comment]
Hi, would you mind providing more information? For example declarative configuration. Be careful to remove private information when providing information.

**Reply by subalalithafl** [reply]
Thanks for the reply. I am just using basic settings. I have changed the conf file with the working folder path and provided POSTGRES DB details. Nothing else is changed in the conf file. For both 2.6 and 2.8 versions.

**Reply by subalalithafl** [reply]
Any help on how to bedug this is also much appreciated. The Key-Auth plugin is also not working. Everything looks same to me.

**Comment by subalalithafl** [comment, marked as answer]
It seems the key-auth plugin from routes was missing. I am not sure how this was removed, but i added them and the plugins are working now. Thanks everyone.
